tests/modules/programs/kodi/example-addon-settings.nix

```
@@ -0,0 +1,22 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  config = {
+    programs.kodi = {
+      enable = true;
+      package = pkgs.writeScriptBin "dummy-kodi" "";
```

There is now a special function to generate stub packages:

```suggestion
      package = config.lib.test.mkStubPackage { };;
```